README.md

```diff
@@ -136,10 +136,22 @@ Provisioning script setup:
 
 Examples:
 
-- `python build_tools/provision.py --runner-id 14474448215 --amdgpu-family gfx94X-dcgpu`: Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 to the default output directory `therock-build`
+- `python build_tools/provision.py --run-id 14474448215 --amdgpu-family gfx94X-dcgpu`: Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 to the default output directory `therock-build`
 
 <br/>
 
-- `python build_tools/provision.py --release-id nightly-release --amdgpu-family gfx110X-dgpu --output-dir build`: Downloads the most recent uploaded gfx110X artifacts from GitHub release tag `nightly-releases` to the specified output directory `build`
+- `python build_tools/provision.py --release-tag nightly-release --nightly-version latest --amdgpu-family gfx110X-dgpu --output-dir build`: Downloads the latest gfx110X artifacts from GitHub release tag `nightly-release` to the specified output directory `build`
+
+<br/>
+
+- `python build_tools/provision.py --release-tag 1.2.3 --amdgpu-family gfx110X-dgpu`: Downloads the gfx110X artifacts from GitHub release tag `1.2.3` to the default output directory `therock-build`
```

Comment:
I am not sure if we will ever have pure `X.Y.Z` release tags as this would correspond to an official ROCm release.